static/stylesheets/application.css

```diff
@@ -6023,3 +6027,102 @@
 #menuToggle:checked ~ .right-menu-info-wrap:before {
   transform: rotate(180deg);
 }
+
+select.form-control {
+  width: 100%;
+  height: 2em;
+}
+
+.form-control {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  font-family: 'nta', Arial, sans-serif;
```

Comment:
new govuk uses `GDS Transport', Arial, sans-serif`.

Reply:
This is generated.  There's a gitignore on this css file that has failed.  I'll check for any examples in the sass file but they're most likely pending removal.

Reply:
Line 146 application.scss.